src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1440,7 +1440,7 @@ public override BoundNode VisitNullCoalescingOperator(BoundNullCoalescingOperato
             }
 
             var leftState = this.State.Clone();
-            var leftResult = leftState.Result;
+            var leftResult = _result;
```

Comment:
Can `leftResult` be declared above with `var leftResult = VisitRvalue(leftOperand);`?

Reply:
Yes. Done.